server/app/views/templates/library/index.scala.html

```diff
@@ -105,7 +105,7 @@ <h2 class="pull-left">@section.name.humanizeUnderscore</h2>
                 @templates.library.contributors(contributors)
 
                 <ul>
-                  <li> <a href="https://github.com/@library.owner/@library.repository/edit/master@section.path.getOrElse("")" target="_blank"><i class="fa fa-pencil"></i>Edit exercises</a>
+                  <li> <a href="https://github.com/@library.owner/@library.repository/edit/main@section.path.getOrElse("")" target="_blank"><i class="fa fa-pencil"></i>Edit exercises</a>
```

Comment:
Thanks @xhudik ,  Could you please also update [line 88](https://github.com/scala-exercises/scala-exercises/pull/987/files#diff-d5fc76c6e4e44c276836efa3711c274938f6b7e8f8ea23baa59e426c5455e8b6R88)?

Reply:
sure, I can and I did. 
I'm not really familiar with this template just wondering - are you sure about this 88 line?  It doesnt look like `edit` page